src/libraries/System.Formats.Tar/src/System/Formats/Tar/PaxTarEntry.cs

```diff
@@ -53,13 +40,10 @@ public PaxTarEntry(TarEntryType entryType, string entryName)
             : base(entryType, entryName, TarEntryFormat.Pax, isGea: false)
         {
             _header._prefix = string.Empty;
-
-            Debug.Assert(_header._mTime != default);
-            AddNewAccessAndChangeTimestampsIfNotExist(useMTime: true);
```

Comment:
If I understand correctly, this was previously adding these times, even if they weren't present.  I agree with this change - we should prioritize preserving the entry in it's original format.  This might be seen as a breaking change to what we did before, but I support this as it's prioritizing round-tripping.